musica/__init__.py

```diff
@@ -2,7 +2,10 @@
 MUSICA: A Python library for atmospheric chemistry simulations.
 """
 
-from ._version import version as __version__
+try:
+    from ._version import version as __version__
+except ImportError:
+    __version__ = "0.12.0"
```

Comment:
```suggestion
try:
    from ._version import version as __version__
except ImportError:
    __version__ = "0.12.0"
```

Revert this change. This already works